upstream_utils/upstream_utils.py

```diff
@@ -243,7 +243,7 @@ def open_repo(self, *, err_msg_if_absent):
 
         if not os.path.exists(dest):
             if err_msg_if_absent is None:
-                subprocess.run(["git", "clone", "--filter=tree:0", self.url])
+                subprocess.run(["git", "clone", self.url])
```

Comment:
Why did we make this change?

Reply:
I committed that on accident. I've been doing that locally because LLVM is way too big for `--filter=tree:0` to work well. It ends up spending half an hour downloading individual pieces of the Git repo during rebases.